### ISSUE TITLE
cssmin: vendor target

### ISSUE THREAD

**Issue by ctalkington** [FlagshipWP/flagship-wp-theme-config#4]
i noticed while adapting a few projects that the vendor target of the cssmin task seems like it would never really run?

looking at the below, this task is normally run before the copy from `tmp/` to `css/` happens. i dont think any of the common tasks actually write to this directory in time for this target to be effective. also, since `css/` is routinely cleaned, you wouldn't want to put any source files in here.

am i missing something obvious?

```json
{
  vendor: {
    expand: true,
    cwd: 'css/',
    src: [
      '*.css'
    ],
    dest: 'css/',
    ext: '.min.css'
  }
```

**Comment by robneu** [comment]
No, I think you're right about that. I'm not sure if this is an issue that was always present or a regression that happened when splitting out the config from the main theme.

**Comment by ctalkington** [comment]
i think if anything, it should almost read from `assets/theme/css` or such for custom css files that are outside of normal process, ie paid fonts that have dumb css required for tracking etc.
